serverless/containers/how-to/secure-a-container.mdx

```diff
@@ -0,0 +1,115 @@
+---
+meta:
+  title: How to secure a container
+  description: This page explains how to secure a Serverless container
+content:
+  h1: How to secure a container
+  paragraph: This page explains how to  how to secure a Serverless container
+tags: containers secure private
+dates:
+  validation: 2023-02-01
+  posted: 2023-02-01
+categories:
+  - serverless
+---
+
+This page explains how to secure your container.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created a containers namespace](/serverless/containers/how-to/create-a-containers-namespace/)
+  - You have [deployed a container](/serverless/containers/how-to/deploy-a-container/)
+</Message>
+
+## Use Secrets to store access keys and sensible information
+
+Instead of using environment variables (which are stored in clear) for your containers, use Secrets. These are information usable through environment variables but are stored encrypted. 
```

Comment:
```suggestion
Instead of using environment variables (which are stored in clear text) for your containers, use Secrets. These are pieces of information that can be used via environment variables, but are encrypted in storage. 
```